### PR TITLE
Update detect-and-remediate-illicit-consent-grants.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to viewing audit logs for Consent To Application as it is not available in Security and compliance any more and can be found in Azure Ad Audit logs.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3801

File: microsoft-365/security/office-365-security/detect-and-remediate-illicit-consent-grants.md [hunk 1]
```diff
@@ -40,15 +40,13 @@ You need to search the **audit log** to find signs, also called Indicators of Co
 
 ### Steps for finding signs of this attack
 
-1. Open the **Security & Compliance Center** at <https://protection.office.com>.
+1. Open the **Azure Active directory** at <https://portal.azure.com/#blade/Microsoft_AAD_IAM/ActiveDirectoryMenuBlade/Overview>.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/detect-and-remediate-illicit-consent-grants.md]:
```suggestion
1. Open [Azure Active directory](https://portal.azure.com/#blade/Microsoft_AAD_IAM/ActiveDirectoryMenuBlade/Overview).
```

File: microsoft-365/security/office-365-security/detect-and-remediate-illicit-consent-grants.md [hunk 2]
```diff
@@ -40,15 +40,13 @@ You need to search the **audit log** to find signs, also called Indicators of Co
...
 
-2. Navigate to **Search** and select **Audit log search**.
+2. Navigate to **Enterprise Applications** and select **Audit logs**.
 
-3. Search (all activities and all users) and enter the start date and end date if required and then click **Search**.
+3. Search (all activities and all users) and enter the  date andthen click **Apply**.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/detect-and-remediate-illicit-consent-grants.md]:
```suggestion
3. Search (all activities and all users) and enter the  date and then click **Apply**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Coppy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MSFTTracyP Please merge this PR. Thanks.

**Comment by yogkumgit** [comment]
Gentle reminder, @MSFTTracyP Please merge this PR. Thanks.

**Pull request merged**